docs/linux.md

```diff
@@ -0,0 +1,107 @@
+# For Linux
+
+## Requirements
+
+1. Git
+2. Docker
+3. Docker Compose
+
+## Install
+
+1. Add the following to your `/etc/hosts` file:
+
+    ```
+    # Redmine
+    0.0.0.0		redmine.local
+    ```
+
+2. Add the following to your `~/.bashrc` file:
+
+    ```bash
+    export HOST_USER_UID=`id -u`
+    export HOST_USER_GID=`id -g`
+
+    alias docker-exec='docker exec -it -u `id -u`:`id -g`'
+    alias docker-tail='docker logs -f --tail=100'
+    ```
+
+3. Source the new `~/.bashrc` file:
+
+    ```bash
+    source ~/.bash_profile
+    ```
+
+4. Clone this repository:
+
+    ``` bash
+    git clone https://github.com/marius-balteanu/redmine_docker.git
+    ```
+
+5. Go inside the `redmine_docker/project` directory and clone the redmine repository:
```

Comment:
It's not clear where is the `redmine_docker/project` directory. It should be created or already exists?